README.rst

```diff
@@ -367,7 +367,7 @@ Contribute
 
 If you spot a design change (or something else) that makes the scraper unable to work for a given site - please fire an issue asap.
 
-If you are programmer PRs with fixes are warmly welcomed and acknowledged with a virtual beer.
+If you are programmer PRs with fixes are warmly welcomed and acknowledged with a virtual beer. You can find documentation on how to develop scrapers `here <https://github.com/hhursev/recipe-scrapers/tree/developer-guide-docs/docs>`_.
```

Comment:
I think a `blob` URL (like the second hyperlink) is the correct one to go for.  A `tree` URL is to an open branch (branch name `developer-guide-docs`) in this case, which may no longer exist / could be cleaned up after the guide is merged.

Reply:
Good catch. 
Relative URLs might have been simpler instead of trying to guess what the URL will be once merged, but I think this willl work.